app/dashboard/migrations/0160_auto_20201111_0445.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.4 on 2020-11-11 04:45
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
hmm this migration shouldnt be on the PR i dont think. nothing to do with the logic of this PR